advisories/a75758.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Technical Advisory 75758
+summary: Users without the appropriate permissions may cancel any other users' sessions from the DB Console
+toc: true
+docs_area: releases 
+---
+
+Publication date: February 9, 2022
+
+## Description
+
+Under some circumstances, all users, including users without the [`admin` role](../v21.2/ui-overview.html#db-console-access) or [`CANCELQUERY` option](../v21.2/create-user.html#role-options), are able to cancel any other users' sessions on the [**Sessions** page](../v21.2/ui-sessions-page.html) of the DB Console on CockroachDB v20.2, v21.1, and v21.2.
+
+This issue is fixed in [v20.2.19](../releases/v20.2.19.html), [v21.1.14](../releases/v21.1.14.html), and [v21.2.5](../releases/v21.2.5.html).
+
+## Statement
+
+This issue is resolved in CockroachDB by [#75814](https://github.com/cockroachdb/cockroach/pull/75814). The fix has been applied to maintenance versions [v20.2.19](../releases/v20.2.19.html), [v21.1.14](../releases/v21.1.14.html), and [v21.2.5](../releases/v21.2.5.html) of CockroachDB. 
+
+This public issue is tracked as [#75758](https://github.com/cockroachdb/cockroach/issues/75758).
+
+## Mitigation
+
+Users of CockroachDB are encouraged to upgrade to a maintenance version with the fix applied: [v20.2.19](../releases/v20.2.19.html), [v21.1.14](../releases/v21.1.14.html), or [v21.2.5](../releases/v21.2.5.html).
+
+## Impact
+
+All deployments up to v20.2.18, v21.1.13, and v21.2.4 are affected.
```

Comment:
Note: This is an addition I made, so the version numbers could use a fact-check.

Reply:
cc @kevin-v-ngo @nickelnickel on this

Reply:
cc @matthewtodd, since you opened #75814. Can you help me confirm that the above versions are the final affected releases for this bug?